server/aws-lsp-codewhisperer/src/index.ts

```diff
@@ -1,5 +1,6 @@
 export * from './language-server/codeWhispererSecurityScanServer'
 export * from './language-server/codeWhispererServer'
 export * from './language-server/qChatServer'
+export * from './language-server/agenticChat/qAgenticChatServer'
```

Comment:
off-topic: this means any symbols in any of these modules may conflict with each other. E.g. symbols from `qAgenticChatServer` may conflict with symbols from `qChatServer`. That then requires every symbol to do its own pseudo-namespacing. 

Instead, index.ts files can do this:

```
export * qAgenticChatServer from './language-server/agenticChat/qAgenticChatServer'
```

and that avoids confilcts and pseudo-namespacing, by using an actual namespace.